music21/test/musicXMLTest.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+import unittest
+from xml.etree import ElementTree as ET
+
+import music21 as m21
+
+DATA_FILE = Path(__file__).parent / 'data/Test1.mxl'
```

Comment:
`xmlToM21` already has a `Test` class, so I would add these three methods there unless @mscuthbert is of the opinion that class is getting too large.

also if your external file is reasonably small (not a bad idea anyway!) you can just block quote it as text in `musicxml.testPrimitive`. otherwise there are .xml and .mxl files loose in the `musicxml` directory which you could add this to, without creating a new `music21.test.data` directory.

figured this would keep this easy to understand for the next person who comes after us!